dev_tools/autogenerate-bloqs-notebooks-v2.py

```diff
@@ -50,7 +50,7 @@
 
 from qualtran_dev_tools.bloq_finder import get_bloqdocspecs
 from qualtran_dev_tools.git_tools import get_git_root
-from qualtran_dev_tools.jupyter_autogen_v2 import NotebookSpecV2, render_notebook
+from qualtran_dev_tools.jupyter_autogen import NotebookSpecV2, render_notebook
```

Comment:
Should we also do NotebookSpecV2 -> NotebookSpec change, or do you want to keep it this way?

Reply:
i'll do that in a follow-up so I don't create any merge conflicts with in-flight things